README.md

```diff
@@ -20,98 +17,26 @@ To run this CLI application
-
-```
-npx create-eth@latest
-```
-
-> 💬 Hint: If you choose Foundry as solidity framework in the CLI, you'll also need Foundryup installed in your machine. Checkout: [getfoundry.sh](https://getfoundry.sh)
-
-2. Run a local network in the first terminal:
-
-```
-yarn chain
-```
-
-This command starts a local Ethereum network using Hardhat or Foundry, depending on which one you selected in the CLI. The network runs on your local machine and can be used for testing and development. You can customize the network configuration in:
-
-- `packages/hardhat/hardhat.config.ts` if you have Hardhat as solidity framework.
-- `packages/foundry/foundry.toml` if you have Foundry as solidity framework.
-
-3. On a second terminal, deploy the test contract:
-
-```
-yarn deploy
-```
-
-This command deploys a test smart contract to the local network. The contract can be modified to suit your needs. Is located in:
-
-- Hardhat => `packages/hardhat/contracts`
-- Foundry => `packages/foundry/contracts`
-
-The `yarn deploy` command uses a deploy script to deploy the contract to the network. You can customize it. Is located in:
-
-- Hardhat => `packages/hardhat/deploy`
-- Foundry => `packages/foundry/script`
-
-4. On a third terminal, start your NextJS app:
-
-```
-yarn start
-```
-
-Visit your app on: `http://localhost:3000`. You can interact with your smart contract using the `Debug Contracts` page. You can tweak the app config in `packages/nextjs/scaffold.config.ts`.
-
-Run smart contract test with `yarn hardhat:test` or `yarn foundry:test` depending of your solidity framework.
-
-**What's next**:
-
-- Edit your smart contract `YourContract.sol` in `packages/hardhat/contracts`
-- Edit your frontend homepage at `packages/nextjs/app/page.tsx`. For guidance on [routing](https://nextjs.org/docs/app/building-your-application/routing/defining-routes) and configuring [pages/layouts](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts) checkout the Next.js documentation.
-- Edit your deployment scripts in `packages/hardhat/deploy`
-- Edit your smart contract test in: `packages/hardhat/test`. To run test use `yarn hardhat:test`
-
-## Documentation
-
-Visit our [docs](https://docs.scaffoldeth.io) to learn how to start building with Scaffold-ETH 2.
-
-To know more about its features, check out our [website](https://scaffoldeth.io).
-
-## Contributing to Scaffold-ETH 2
-
-We welcome contributions to Scaffold-ETH 2!
-
-Please see [CONTRIBUTING.MD](https://github.com/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) for more information and guidelines for contributing to Scaffold-ETH 2.
+## MVP
+- ETT CLI enables users to see the full scope of the tech tree
+- They can see clearly which ones are unlocked and which ones are still locked
+- It only allows them to interact with the challenges that are unlocked
+- It shows them their progress/general score on each challenge and total score on each branch of the tree
```

Comment:
So it would be a pairing of XP points, and pre-reqs for specific challenges (let's say the DeFi challenges require either some DeFi Personal Challenge to be passed, or that they get some key DeFi specific challenges done in succession).

Reply:
The XP idea is something in the future section. Rewording this to be clearer.

Matt had the idea that we grade their submitted challenges based on tests that pass. Instead of being a simple PASS/FAIL it would be PASS/FAIL for some basic functionality (maybe 60% of their grade) but then additional tests would also need to pass to get a higher score. Love this idea but it also requires writing a lot of really creative tests and building the grader to handle it all, hence it is future.